packages/ui/src/domainModules/agent/components/AgentManager/item/BaseForm/unitSpecs/agentNameDetails/index.js

```diff
@@ -55,6 +55,17 @@ const parts = [
     name: 'alsoKnownAs',
     wrapInField: true,
   },
+  {
+    componentName: 'Input',
+    componentProps: {
+      columnProps: { width: 16 },
+      style: { width: '18em' },
```

Comment:
we should make a general solution for this case, when they want to have small columns stacked and not in the same row

Reply:
I added two commits to fix this, please have a look